detox/ios/Detox/MethodInvocation.m

```diff
@@ -80,6 +80,24 @@ + (id) getValue:(id)value withType:(id)type onError:(void (^)(NSString*))onError
         if (![value isKindOfClass:[NSDictionary class]]) return nil;
         return [MethodInvocation invoke:value onError:onError];
     }
+	if ([type isEqualToString:@"UIDeviceOrientation"])
```

Comment:
Does this mean I could have written `invoke.IOS.UIDeviceOrientation('UIDeviceOrientationLandscapeLeft')` or what would the JS call need to look like in this case?